src/containerapp/azext_containerapp/containerapp_sessionpool_decorator.py

```diff
@@ -42,6 +42,7 @@
 class ContainerType(Enum):
     PythonLTS = 0
     CustomContainer = 2
+    NodeLTS = 3
```

Comment:
Does it work with only add type here?

Reply:
hi @ASHIQUEMD to make the new container type actually be sent in the ARM you also need to be making changes to the src/containerapp/azext_containerapp/containerapp_session_code_interpreter_decorator.py
file to pass this new property in, you can refer to my PR below to see an example and also how to add tests. let me know if you have any questions :)

https://github.com/Azure/azure-cli-extensions/pull/7607

Reply:
Hi @smzeng, in terms of code execution, the container type will be derived from the session pool name, so I believe no changes are needed in that area. The container type is indeed required for creating the session pool, and this has already been added to containerapp_sessionpool_decorator.py.

Please let me know if I've misunderstood anything or if there's anything else you'd like to discuss.

Reply:
Hi @ASHIQUEMD 
For `in terms of code execution, the container type will be derived from the session pool name, so I believe no changes are needed in that area.` I can not find where is the code, can you show me where it is?

Reply:
@Greedygre this implementation is present in [ACA session data plane code](https://msazure.visualstudio.com/One/_git/AAPT-Antares-Containerapps?path=/src/Microsoft.ContainerApps/Microsoft.ContainerApps.Services/Services/Sessions/PythonExecutionApiService.cs&version=GBmain&line=599&lineEnd=611&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents), where based on the passed `sessionPoolName` we read the type of container.

Reply:
Got it, thanks